src/components/data_submission/DataAccessGovernance.js

```diff
@@ -40,22 +94,92 @@ export const DataAccessGovernance = (props) => {
   const addNewConsentGroup = useCallback(() => {
     setConsentGroupsState((consentGroupsState) => {
       const newConsentGroupsState = cloneDeep(consentGroupsState);
-      newConsentGroupsState.push({
+      newConsentGroupsState?.push({
```

Comment:
```suggestion
      newConsentGroupsState.push({
```

Since on L176 you check if consentGroupState is empty, this should never be null or undefined.